chapter1/README.md

```diff
@@ -1,4 +1,22 @@
-# ManageIQ 
+# Walk to the Chapter
 
-About this chapter: one para
+This Chapter will talk about every bit and byte of ManageIQ. It will serve as guidelines for all those folks who wish to start with it. The Chapter starts with the concept of ManageIQ and ends with its full installation. It's an kind request to all the readers that they make sure the mentioned recommendations of infrastructure are properly met unless otherwise the system may hang completely.
```

Comment:
- 'c' in the word 'chapter' should be small. 
- It's a(not an) kind request ..  is correct.